.github/workflows/budibase_ci.yml

```diff
@@ -44,7 +44,10 @@ jobs:
           node-version: 14.x
           cache: "yarn"
       - run: yarn
-      - run: yarn nx run-many -t=build --configuration=production
+      # Run build all the projects
+      - run: yarn nx run-many -t=build
```

Comment:
`types`, `backend-core` etc, looks like they still use `tsc`. Does that mean we'll still have potentially slower build times on those packages?

Reply:
Yes, these packages still use tsc, so we have the same performance as before. The `configuration=production` has only effect if using `nx/esbuild`, for this I removed it into the package itself as the pipeline should not be aware of this things, otherwise we will run into issues. I actually already got some issues that I missed adding the checks to `server`, as I needed to remove the `nx/esbuild` from there because of vite.

Reply:
Yes, we will be moving them over now that everything works fine. The preference was worker and server, because doing this we can now achieve npm-less docker images